docs/walkthrough_search.md

```diff
@@ -3,15 +3,51 @@ title: Search in eScriptorium
 summary: About the search feature in eScriptorium.
 authors:
     - Alix Chagué
-date: 2023-01-31
+date: 2023-04-19
 ---
 
-# Walkthrough : search in eScriptorium
+# Walkthrough: search in eScriptorium
 
 !!! warning
     The section needs to be completed.
 
 ## Activate search
```

Comment:
Since you are using a capital "S" for search in the subsequent paragraph, maybe you could put it in the title too ?